deployment/common/changeset/internal/mcms.go

```diff
@@ -0,0 +1,137 @@
+package internal
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/config"
+	owner_helpers "github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/deployment/common/view/v1_0"
+)
+
+func DeployMCMSWithConfig(
```

Comment:
Intend to follow up with the same pattern here for CCIP dir. Deployment logic can be internal to enforce changeset API usage in higher level tests